### ISSUE TITLE
salt.modules.runit doesn't allow management of log services.

### ISSUE THREAD

**Issue by dragonpaw** [saltstack/salt#25382]
Running under runit, every directory spins up usually 2 services, not 1. There's a /log service also for the vast majority of runit services.

Which leads to fun like this:

``` console
# salt-call runit.status socklog-unix/log
[INFO    ] Executing command 'sv status /etc/service/socklog-unix/log' in directory '/home/ubuntu'
local:
    13497
# salt-call runit.available socklog-unix/log
local:
    False
```

Can be fixed by changing `salt.modules.runit` like so:

``` python
def get_all():
    '''
    Return a list of all available services

    CLI Example:

    .. code-block:: bash

        salt '*' runit.get_all
    '''
    if not SERVICE_DIR:
        raise CommandExecutionError("Could not find service directory.")

    services = []
    for s in os.listdir(SERVICE_DIR):
        services.append(s)
        if os.path.exists(os.path.join(SERVICE_DIR, s, 'log')):
            services.append(s + '/log')
    return sorted(services)
```

**Comment by jfindlay** [comment]
@dragonpaw, you are welcome to submit your fix to the 2015.5 branch.  Thanks for reporting.

**Comment by dragonpaw** [comment]
It's still broken.
